src/condor_tests/test_curl_plugin.py

```diff
@@ -85,13 +91,28 @@ def job_with_good_url(default_condor, good_url, test_dir, path_to_sleep):
     assert job.wait(condition=ClusterState.all_terminal)
     return job
 
+@action
+def job_with_good_file_url(default_condor, good_file_url, test_dir, path_to_sleep):
+    job = default_condor.submit(
+        {
+            "executable": path_to_sleep,
+            "arguments": "0",
+            "log": (test_dir / "good_file_url.log").as_posix(),
+            "transfer_input_files": good_file_url,
+            "transfer_output_files": "file_url_tester",
+            "should_transfer_files": "YES",
+            "requirements": "(SlotID == 1)"
```

Comment:
Why only slot 1?

Reply:
Because the other tests that test successful file xfer use this slot.